.travis.yml

```diff
@@ -216,10 +217,10 @@ matrix:
     ### Python Tests ###
     - &pytools-vm
       stage: "Pytest"
-      name: "tools-py27"
-      env: NAME=tools-py2.7
+      name: "tools-py35"
+      env: NAME=tools-py3.5
       language: python
-      python: 2.7
+      python: 3.5
```

Comment:
Python 3.5 has also reached end-of-life. Since it's a recent event (late 2020), it's okay to keep or remove it I guess.